src/main/resources/db/migration/V0.0.2_002__create_assessment_quest.sql

```diff
@@ -0,0 +1,86 @@
+   create table assessment_questionnaire (
+       id int8 not null,
+        createTime timestamp,
+        createUser varchar(255),
+        deleted boolean not null default false,
+        updateTime timestamp,
+        updateUser varchar(255),
+        language_code varchar(255) not null,
+        name varchar(255) not null,
+        assessment_id int8 not null,
+        questionnaire_id int8 not null,
+        primary key (id)
+    );
+    
+    create table assessment_category (
```

Comment:
I can not see any field for saving the user comments. The UI mockups suggest having 1 comment per category.

Reply:
That has been added in the next PR for the GET opperation.